<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldEdit</class>
 <widget class="QMainWindow" name="WorldEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>7</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="SceneView" name="sceneView" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLight_tool"/>
   <addaction name="actionTerrain_tool"/>
   <addaction name="actionActor_tool"/>
   <addaction name="actionToggleBones"/>
   <addaction name="actionToggleBoundingBoxes"/>
  </widget>
  <widget class="QDockWidget" name="terrainToolDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="lightToolDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="actorEditToolDock">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="fogToolDock">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <widget class="QDockWidget" name="godrayToolDock">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5"/>
  </widget>
  <action name="actionLight_tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../build/vs10/WtEditor/WtEditor.qrc">
     <normaloff>:/icons/placeholder</normaloff>:/icons/placeholder</iconset>
   </property>
   <property name="text">
    <string>Light tool</string>
   </property>
  </action>
  <action name="actionTerrain_tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../build/vs10/WtEditor/WtEditor.qrc">
     <normaloff>:/icons/placeholder</normaloff>:/icons/placeholder</iconset>
   </property>
   <property name="text">
    <string>Terrain tool</string>
   </property>
  </action>
  <action name="actionActor_tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../build/vs10/WtEditor/WtEditor.qrc">
     <normaloff>:/icons/placeholder</normaloff>:/icons/placeholder</iconset>
   </property>
   <property name="text">
    <string>Actor tool</string>
   </property>
  </action>
  <action name="actionToggleBones">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../build/vs10/WtEditor/WtEditor.qrc">
     <normaloff>:/icons/placeholder</normaloff>:/icons/placeholder</iconset>
   </property>
   <property name="text">
    <string>Toggle Bones</string>
   </property>
  </action>
  <action name="actionToggleBoundingBoxes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../build/vs10/WtEditor/WtEditor.qrc">
     <normaloff>:/icons/placeholder</normaloff>:/icons/placeholder</iconset>
   </property>
   <property name="text">
    <string>Toggle bounding boxes</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SceneView</class>
   <extends>QWidget</extends>
   <header>sceneview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../build/vs10/WtEditor/WtEditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionActor_tool</sender>
   <signal>triggered()</signal>
   <receiver>WorldEdit</receiver>
   <slot>onToggleToolActor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLight_tool</sender>
   <signal>triggered()</signal>
   <receiver>WorldEdit</receiver>
   <slot>onToggleToolLight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTerrain_tool</sender>
   <signal>triggered()</signal>
   <receiver>WorldEdit</receiver>
   <slot>onToggleToolTerrain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleBones</sender>
   <signal>triggered()</signal>
   <receiver>WorldEdit</receiver>
   <slot>onToggleBones()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleBoundingBoxes</sender>
   <signal>triggered()</signal>
   <receiver>WorldEdit</receiver>
   <slot>onToggleBoundingBoxes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onScreenshot()</slot>
  <slot>onClearAll()</slot>
  <slot>onSave()</slot>
  <slot>onSetSkybox()</slot>
  <slot>onCreateTerrain()</slot>
  <slot>onToggleToolLight()</slot>
  <slot>onToggleToolTerrain()</slot>
  <slot>onToggleToolActor()</slot>
  <slot>onToggleBoundingBoxes()</slot>
  <slot>onToggleBones()</slot>
 </slots>
</ui>
